.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
They use arm toolchain to cross-compile for the target board (RPI, BBB, stm32 etc). It doesn't make sense to build that targets with the compiler tsegetting the development system.

If there are clang cross compilers for these platforms then we can use them